packages/rum-core/src/common/service-factory.js

```diff
@@ -42,7 +42,9 @@ class ServiceFactory {
       return new ConfigService()
     })
     this.registerServiceCreator('LoggingService', function() {
-      return new LoggingService()
+      return new LoggingService({
+        prefix: '[JS Agent] '
```

Comment:
What do you think about using `Elastic APM` here instead?

Reply:
Yeah sounds good.

Reply:
Done.